src/test/java/org/fundacionjala/coding/omar/encryption/EncryptionTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * @(#)EncryptionTest.java
+ *
+ * Copyright (c) 2018 Jala Foundation.
+ * Address
+ * All rights reserved.
+ *
+ * This software is the confidential and propietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Sun.
+ */
+package org.fundacionjala.coding.omar.encryption;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class contains UnitTest to test CesarEncryption class.
+ *
+ * @author Omar Limbert Huanca Sanchez - AT-[06].
+ * @version 1.0.
+ */
+public class EncryptionTest {
+
+    private static final String VIGENERE_KEY = "LIMON";
+    private CesarEncryption cesarEncryption;
+    private VigenereEncryption vigenereEncryption;
+
+    /**
+     * init Kata.
+     */
+    @Before
+    public void init() {
+
+        cesarEncryption = new CesarEncryption();
+        vigenereEncryption = new VigenereEncryption();
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class